src/loaders/svg/tvgSvgPath.cpp

```diff
@@ -499,11 +499,8 @@ static char* _nextCommand(char* path, char* cmd, float* arr, int* count)
 }
 
 
-tuple<vector<PathCommand>, vector<Point>> svgPathToTvgPath(const char* svgPath)
+bool svgPathToTvgPath(const char* svgPath, SvgVector<PathCommand>& cmds, SvgVector<Point>& pts)
```

Comment:
The return value from this function is always true. Failure cases are not yet checked. I think it is right to change it to void type.